tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -211,11 +211,11 @@ describe('Service Form Modal', function () {
       killSelection: 'YOUNGEST_FIRST',
       portDefinitions: [
         {
-          port: 10000,
+          port: 0,
           protocol: 'tcp'
         }
       ],
-      requirePorts: true
+      requirePorts: false
```

Comment:
We should never change `requirePorts` basing on a port value,
in `_fixtures/marathon-1-task-with-columes/groups.json`
we see 
```
{
...
      "ports": [
        10000
      ],
      "portDefinitions": [
        {
          "port": 10000,
          "protocol": "tcp",
          "labels": {
            
          }
        }
      ],
      "requirePorts": false,
...
}
```

Port value 10000 has been auto assigned because `"requirePorts": false` and Marathon will return it.
One shouldn't even take into consideration those ports and use env vars $PORT0...$PORT<n>